frontend/src/components/WofEditPopup.vue

```diff
@@ -0,0 +1,123 @@
+<!-- 
+Component changes something.
+Example:
+<wof-edit-popup title="Edit username" :visible="true" currentValueName="Username" currentValue="wofUser123"/>
+-->
+
+
+<template>
+    <wof-info-box class="wof-edit-popup" :title="this.title" :isOpen="this.state" v-if="this.state"  @close="changeState">
+
```

Comment:
Wrap those "wof-edit-popup__field" and "wof-edit-popup__buttons" in one div with class named like 'wof-edit-popup__body' or something.
Move div with loading next to this.

wof-edit-popup__body should have visibility: hidden if loading is true (add computed property for that)
if loading is true, wof-spinner-dots should visible and displayed in the center of this popup.